host_modules/systemd_service.py

```diff
@@ -7,6 +7,8 @@
 ALLOWED_SERVICES = ['snmp', 'swss', 'dhcp_relay', 'radv', 'restapi', 'lldp', 'sshd', 'pmon', 'rsyslog', 'telemetry']
 EXIT_FAILURE = 1
 
+REBOOTMETHOD_COLD_BOOT_VALUES = {1, "COLD"}
+REBOOTMETHOD_HALT_BOOT_VALUES = {3, "HALT"}
```

Comment:
@vvolam where are these values coming from?

Reply:
@prgeor They are defined in https://github.com/openconfig/gnoi/blob/main/system/system.proto

Reply:
@vvolam Could you just add this reference link as code comment? it will help future dev or yourself.